apps/showcase/project.json

```diff
@@ -30,6 +30,7 @@
         "script": "postbuild:patch"
       },
       "dependsOn": [
+        "^build",
```

Comment:
Not needed here, it is part of the `build-source`.
I also think that the `build-source` default dependency from `nx.json` is already including the `^build`